tfjs-core/src/platforms/platform_browser_test.ts

```diff
@@ -88,3 +89,62 @@ describeWithFlags('PlatformBrowser', BROWSER_ENVS, async () => {
     expect(s).toEqual('Здраво');
   });
 });
+
+describeWithFlags('setTimeout', BROWSER_ENVS, () => {
+  const totalCount = 100;
+  // Skip the first few samples because the browser does not clamp the timeout
+  const skipCount = 5;
+
+  it('setTimeout', (done) => {
```

Comment:
@gyagp I began to see this test fail on Linux, maybe related with https://chromestatus.com/feature/5710690097561600

Reply:
@hujiajie What's your channel and version of Chrome? I tried both  Stable (108.0.5359.124) and Dev (110.0.5478.4) on Linux,  but the nesting levels are still 5. I also double checked Windows, with Stable (108.0.5359.125) and Canary (111.0.5491.0), and the nesting levels are also 5.
My testing program is timer.html at https://github.com/webatintel/webatintel.github.io. You need to download it to local (otherwise the result is impacted by the network condition) and type "<pathToScript>/timer.html?test=setTimeout&maxCount=30" in the address bar.

Reply:
Ah, it seems chromestatus is lying here, and the feature is still in field trial. What if you add `--enable-features=MaxUnthrottledTimeoutNestingLevel`?